manifests/istio-telemetry/tracing/values.yaml

```diff
@@ -35,7 +35,7 @@ tracing:
   jaeger:
     hub: docker.io/jaegertracing
     image: all-in-one
-    tag: 1.14
+    tag: "1.14"
```

Comment:
if you are making this change because it breaks istioctl, that means you will probably break real users as well

Reply:
these are already broken for users, but it's a good idea to fix this now rather than later.

Reply:
Done.